### ISSUE TITLE
NOTES

### ISSUE THREAD

**Issue by tommygebru** [egad1868/prj-rev-bwfs-dasmoto#7]
_NOTE: Always conduct a peer-review with your peers in Slack, this will improve performance and ability to problem solve!_ 💯

Your project contains little to no errors Amy, I went ahead and reviewed your work for best practices and general areas of improvement 😃

**Comment by egad1868** [comment]
You say go for peer reviews, that makes me a little nervous as we are all here learning and what if we give each other incorrect answers? In fact that has caused me some "pain" in my learning in the past with a lot of trial and error and time with incorrect information. Additionally as you know there are only best practices and many people have "their" way of writing code that is just a little different but still valid. So what is the best way to leverage peer reviews in your opinion without causing or getting more problems? If it's not obvious I've not worked in a collaborative or supportive environment before this. :/

**Comment by tommygebru** [comment]
Thats true, there are many opinions and resources out there, this requires you to be vigilant and trying things out for yourself. You cannot avoid errors or mistakes when coding, you can only "repeat and learn" towards a set goal or outcome. So always ask questions or help, read articles and enjoy improving your work or process. 👍
